tests/core/test_feature.py

```diff
@@ -160,6 +173,44 @@ def test_feature_from_df(df):
 
     # clean up
     artifact.delete(permanent=True)
-    ln.Schema.filter().all().delete()
-    ln.ULabel.filter().all().delete()
-    ln.Feature.filter().all().delete()
+    ln.Schema.filter().delete()
+    ln.ULabel.filter().delete()
+    ln.Feature.filter().delete()
+
+
+def test_feature_from_dict(dict_data):
+    # ambiguous str types
+    with pytest.raises(ValueError) as e:
+        features = ln.Feature.from_dict(dict_data, str_as_cat=None)
+    error_msg = str(e.value)
+    assert "Ambiguous dtypes detected" in error_msg
+    assert "'dict_feat3': str or cat" in error_msg
+    assert "'dict_feat6': list[str] or list[cat]" in error_msg
+    assert "Please specify `str_as_cat` parameter" in error_msg
```

Comment:
We typically say "pass" rather than "specify"

"passing an argument to a function"